invesalius/gui/network/nodes_panel.py

```diff
@@ -0,0 +1,306 @@
+from invesalius.gui.language_dialog import ComboBoxLanguage
+from invesalius.pubsub import pub as Publisher
+import invesalius.net.dicom as dcm_net
+import invesalius.constants as const
+import invesalius.session as ses
+import wx.grid as gridlib
+import sys
+import wx
+
+class NodesPanel(wx.Panel):
+    """ Dicom Nodes Dialog. """
+
+    def __init__(self, parent):
+        wx.Panel.__init__(self, parent, -1)
+        
+        self.__nodes = []
+        self.__selected_index = None
+        self.__selected_node = None
+
+        self.__session = ses.Session()
+
+        # Create the main vertical box sizer
+        main_sizer = wx.BoxSizer(wx.VERTICAL)
+
+        self.__find_input = wx.TextCtrl(self, size=(225, -1))
+        self.__find_input.SetHint(_("Enter patient name"))
+        
+        find_sizer = self._create_find_box_sizer()
+
+        buttons_sizer = self._create_buttons_box_sizer()
+
+        hor = wx.BoxSizer(wx.HORIZONTAL)
+
+        hor.Add(find_sizer, 0, wx.EXPAND | wx.ALL, 5)
+        hor.AddStretchSpacer()
+        hor.Add(buttons_sizer, 0, wx.EXPAND | wx.ALL, 5)
+
+        main_sizer.Add(hor, 0, wx.EXPAND)
+
+        self.__list_ctrl = self._create_list_ctrl()
+
+        # Load values from config file
+        self._load_values()
+        
+        # Add the grid table to the main sizer
+        main_sizer.Add(self.__list_ctrl, 1, wx.GROW | wx.EXPAND)
+        
+        # Create the input fields
+        self.__ipaddress_input = wx.TextCtrl(self, size=(225, -1))
+        self.__ipaddress_input.SetHint(_("127.0.0.1"))
+        self.__port_input = wx.TextCtrl(self, size=(225, -1))
+        self.__port_input.SetHint(_("4242"))
+        self.__aetitle_input = wx.TextCtrl(self, size=(225, -1))
+        self.__aetitle_input.SetHint(_("ORTHANC"))
+        self.__description_input = wx.TextCtrl(self, size=(225, -1))
```

Comment:
Ip address, port, aetitle don't need `_("")`. This is only needed for string that can be translated.

Reply:
Oh, i understood. i removed the anotations for these strings that will not be translated

thanks for the tip